app/models/form/sales/pages/handover_date_check.rb

```diff
@@ -6,7 +6,7 @@ def initialize(id, hsh, subsection)
                    { "saledate_check" => 1, "hodate_3_years_or_more_saledate?" => true }]
     @informative_text = {}
     @title_text = {
-      "translation" => "validations.sale_information.hodate.must_be_less_than_3_years_from_saledate",
+      "translation" => "validations.sale_information.hodate.must_be_less_than_3_years_from_saledate_soft",
```

Comment:
I think there is a separate `soft_validations` section (instead of `validations`) in our translations file, would it make sense to move this message there instead of suffixing it?

Reply:
Good spot - done!